build.sh

```diff
@@ -30,7 +30,7 @@ JOBS=""
 CONFIGURATION="RelWithDebInfo"
 REBUILD="no"
 VERBOSE="no"
-USE_ZLIBNG="no"
```

Comment:
Do we need changes to build_windows.bat as well?

Reply:
Nope, the Windows build relies on the defaults in [`CMakeLists.txt`](https://github.com/dotnet/android-libzipsharp/pull/145/files#diff-1e7de1ae2d059d21e1dd75d5812d5a34b0222cef273b7c3a2af62eb747f9d20aR15). The variables in `build.sh` are just for convenience of testing and development on Unix machines.